.openpublishing.redirection.json

```diff
@@ -6,31 +6,31 @@
     },
     {
       "source_path_from_root": "/docs/storage/azure-storage.md",
-      "redirect_url": "/dotnet/aspire/storage/azure-storage-components"
+      "redirect_url": "/dotnet/aspire/storage/azure-storage-integrations"
     },
     {
       "source_path_from_root": "/docs/database/quickstart-sql-server.md",
-      "redirect_url": "/dotnet/aspire/database/sql-server-components"
+      "redirect_url": "/dotnet/aspire/database/sql-server-integrations"
     },
     {
       "source_path_from_root": "/docs/get-started/quickstart-build-your-first-aspire-app.md",
       "redirect_url": "/dotnet/aspire/get-started/build-your-first-aspire-app"
     },
     {
       "source_path_from_root": "/docs/messaging/quickstart-messaging.md",
-      "redirect_url": "/dotnet/aspire/messaging/messaging-components"
+      "redirect_url": "/dotnet/aspire/messaging/messaging-integrations"
     },
     {
       "source_path_from_root": "/docs/caching/quickstart-caching.md",
-      "redirect_url": "/dotnet/aspire/caching/caching-components"
+      "redirect_url": "/dotnet/aspire/caching/caching-integrations"
     },
     {
       "source_path_from_root": "/docs/app-host-overview.md",
       "redirect_url": "/dotnet/aspire/fundamentals/app-host-overview"
     },
     {
       "source_path_from_root": "/docs/components-overview.md",
```

Comment:
is there a path to updating the source paths too?

Reply:
I want to make sure I understand what you're asking here. Are you asking if we're planning on updating any of the file paths as part of this effort too, if so, what changes would you expect? Or are you asking something else?

Reply:
Yes that. I noted the `source_path_from_root` still says "/docs/components-overview.md". Just wondering if we're planning to change that to "/docs/integrations-overview.md".

Reply:
Yeah, our OPS build system relies on DocFX to do change detection in the redirects and we get dinged if we change certain existing rules that were already in place. The file and route did in fact change to integrations, and after this PR is in place the only existence of the "components phrase will be in reference to Dapr components or components in articles that talk about web related bits.